content/configuration/communication/_index.md

```diff
@@ -6,87 +6,236 @@ weight: 10
-                                        # Type is optional and default is short.
-
-  # Settings for Mattermost
-  mattermost:
-    enabled: false
-    url: 'MATTERMOST_SERVER_URL'        # URL where Mattermost is running.
-                                        # e.g https://example.com:9243
-    token: 'MATTERMOST_TOKEN'           # Personal Access token generated by BotKube user
-    team: 'MATTERMOST_TEAM'             # Mattermost Team to configure with BotKube
-    channel: 'MATTERMOST_CHANNEL'       # Mattermost Channel for receiving BotKube alerts
-    botName: 'BotKube'                  # Bot username    
-    notification:
-      type: short                       # Change notification type short/long.
-                                        # Type is optional and default is short.
-
-  # Settings for Microsoft Teams
-  teams:
-    enabled: false
-    appID: 'APPLICATION_ID'
-    appPassword: 'APPLICATION_PASSWORD'
-    botName: 'BotKube'
-    port: 3978
-    notification:
-      type: short                       # Change notification type short/long.
-                                        # Type is optional and default is short.
-
-  # Settings for Discord
-  discord:
-    enabled: false
-    token: 'DISCORD_TOKEN'	            # BotKube Bot Token
-    botID: 'DISCORD_BOT_ID'             # BotKube Application Client ID
-    channel: 'DISCORD_CHANNEL_ID'       # Discord Channel id for receiving BotKube alerts
-    notification:
-      type: short                       # Change notification type short/long.
-                                        # Type is optional and default is short.
-
-  # Settings for ELS
-  elasticsearch:
-    enabled: false
-    awsSigning:
-      enabled: false                    # enable awsSigning using IAM for Elastisearch
-                                        # hosted on AWS, if true make sure AWS
-                                        # environment variables are set.
-                                        # Ref: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
-      awsRegion: 'us-east-1'            # AWS region where Elasticsearch is deployed
-      roleArn: ''                       # AWS IAM Role arn to assume for credentials,
-                                        # use this only if you dont want to
-                                        # use the EC2 instance role or not
-                                        # running on AWS instance
-    server: 'ELASTICSEARCH_ADDRESS'     # e.g https://example.com:9243
-    username: 'ELASTICSEARCH_USERNAME'  # Set creds if using Basic Auth
-    password: 'ELASTICSEARCH_PASSWORD'
-    # ELS index settings
-    index:
-      name: botkube
-      type: botkube-event
-      shards: 1
-      replicas: 0
-
-  # Settings for Webhook
-  webhook:
-    enabled: false
-    url: 'WEBHOOK_URL'                 # e.g https://example.com:80
+  'default-group':
+    ## Settings for Slack.

```

Comment:
**nit**
I know that you have that directly from the `values.yaml` but IMO it would be nice to get rid of `# --` and `##` as in this case it doesn't make sens (It's easy with IntelliJ and multi cursor feature). As a result, it will be consistent with the YAML from the `example` section.

Reply:
I don't think these make it harder to understand the config, but I will do that for consistency 👍 In future we can revisit that though, and maybe do not modify comments to make it easier to synchronize changes.